pkgs/development/python-modules/types-mysqlclient/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+let
+  version = "2.2.0.20250516";
+in
+buildPythonPackage {
+  inherit version;
+  pname = "types-mysqlclient";
+  format = "setuptools";
```

- https://github.com/NixOS/nixpkgs/issues/253154
```suggestion
  pyproject = true;
```